doc/content/integrations/webhooks/webhook-templates/format.md

```diff
@@ -53,5 +53,6 @@ The message paths are provided in the `paths` object which can contain the follo
 - `downlink-queued`: The path to which downlink queued status will be sent. Can contain template fields.
 - `downlink-queue-invalidated`: The path to which downlink queue invalidated event will be sent. Can contain template fields. This is only used when the upstream platform carries out LoRaWAN `FRMPayload` encryption.
 - `location-solved`: The path to which the location of the device will be sent when resolved. Can contain template fields.
+- `service-data`: The path to which the data from services such as LoRaCloud DAS and GLS will be sent. Can contain template fields.
```

Comment:
This is not specific to LoRa Cloud (two words). This is for all service data generated by integrations.